Extension/src/LanguageServer/client.ts

```diff
@@ -877,6 +879,15 @@ class DefaultClient implements Client {
         return this.queueTask(() => Promise.resolve(this.configuration.CompilerPath));
     }
 
+    public getVcpkgInstalled(): Thenable<boolean> {
+        return this.queueTask(() => Promise.resolve(this.configuration.VcpkgInstalled));
+    }
+
+    public getDependencyManager(): Thenable<string> {
+        const cppSettings: CppSettings = new CppSettings(this.RootUri);
+        return this.queueTask(() => Promise.resolve(cppSettings.dependencyManager));
```

Comment:
You don't need to queue this one because it doesn't depend on the configuration.